plugins/techdocs-addons-test-utils/api-report.md

```diff
@@ -0,0 +1,47 @@
+## API Report File for "@backstage/plugin-techdocs-addons-test-utils"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { ApiRef } from '@backstage/core-plugin-api';
+import { default as React_2 } from 'react';
+import { ReactElement } from 'react';
+import { screen as screen_2 } from 'testing-library__dom';
+import { TechDocsEntityMetadata } from '@backstage/plugin-techdocs-react';
+import { TechDocsMetadata } from '@backstage/plugin-techdocs-react';
+
+// @public (undocumented)
+export class TechDocsAddonTester {
+  // (undocumented)
+  atPath(path: string): this;
+  // (undocumented)
+  build(): React_2.ReactElement<
+    any,
+    string | React_2.JSXElementConstructor<any>
+  >;
+  // (undocumented)
+  static buildAddonsInTechDocs(addons: ReactElement[]): TechDocsAddonTester;
+  // (undocumented)
+  renderWithEffects(): Promise<
+    typeof screen_2 & {
+      shadowRoot: ShadowRoot | null;
+    }
+  >;
+  // (undocumented)
+  withApis<T>(apis: TechdocsAddonTesterApis<T>): this;
+  // (undocumented)
+  withDom(dom: ReactElement): this;
+  // (undocumented)
+  withEntity(entity: Partial<TechDocsEntityMetadata>): this;
+  // (undocumented)
+  withMetadata(metadata: Partial<TechDocsMetadata>): this;
+}
+
+// @public (undocumented)
+export type TechdocsAddonTesterApis<T> = TechDocsAddonTesterTestApiPair<T>[];
```

Comment:
Tbh I'd rather we `@ignore` these and don't export them, like in test-utils. We could consider exporting the types from test-utils or something like that if it ends up being a common need.